File: Universal Windows Platform Solution/Phantom of the West/Phantom of the West/Visual Novel/Chapters/Chapter 3/IRefuseToDiguiseAsInfant.cs
using Phantom_of_the_West.Visual_Novel.Serialization.Event_Flagging;

namespace Phantom_of_the_West.Visual_Novel.Chapters.Chapter_3
{
	internal class IRefuseToDiguiseAsInfant : StoryChoice
	{
		internal IRefuseToDiguiseAsInfant() : base(StoryChoice.GetLocalizedString("ch3IRefuseToDiguiseAsInfant"), 3035)
		{
		}

		public override void Select()
		{
			PotWVN vn = PotWVN.MainVN;
			vn.EventFlags.Ch3DroveWithCaitlyn = DriveWithCaitlyn.PossessedCaitlyn;
			base.Select();
		}
	}
}

File: Android Project/app/src/main/java/com/christopherraleigh/phantomofthewest/visual_novel/chapters/chapter_3/IRefuseToDiguiseAsInfant.java
package com.christopherraleigh.phantomofthewest.visual_novel.chapters.chapter_3;

import com.christopherraleigh.phantomofthewest.R;
import com.christopherraleigh.phantomofthewest.visual_novel.PotWVN;
import com.christopherraleigh.phantomofthewest.visual_novel.chapters.StoryChoice;
import com.christopherraleigh.phantomofthewest.visual_novel.serialization.event_flagging.DriveWithCaitlyn;

/**
 * Created by christopherraleigh on 2016-05-12.
 */
public class IRefuseToDiguiseAsInfant extends StoryChoice {
    public IRefuseToDiguiseAsInfant() {
        super(R.string.ch3IRefuseToDiguiseAsInfant, 3035);
    }

    @Override
    public void select() {
        PotWVN vn = PotWVN.getMainVN();
        vn.getEventFlags().setCh3DroveWithCaitlyn(DriveWithCaitlyn.PossessedCaitlyn);
        super.select();
    }
}
